html/templates/category-filter-selectlist.mst

```diff
@@ -2,6 +2,13 @@
    <li class="selectlist">Show indicators sorted/grouped by:
       <div id="js-category-filter-select"></div>
    </li>
+   <li class="filtered-list-message selectoptions alert-warning" style="display: none" ><p id="list-message">You are viewing a filtered list. Click the reset button to view the entire list.</p>
+   <p class="info-message" style="display: none" >The results have been filtered due to some parameters in the URL. A WMS URL or URL name have been specified, and the matching results are searchable below. If you were expecting a layer to load, you must make sure that both of these variables are declared.</p>
```

Comment:
I think that mentioning variables to end users will cause confusion. I would change this to 'The results shown below have been filtered to match your initial request, however, there are more indicators available for display. If you click the Reset button the panel below will reload to display all the available indicators'